_posts/2018-03-02-Kickstarter-Trajectory-2018-light.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+authors: [johan_silkens, nick_van_hoof, sam_schuddinck, dries_thieren, maarten_westelinck, yunus_altin, yen_mertens]
+title: "Kickstarter Trajectory 2018 Light Edition"
+image: /img/kicks.png
+tags: []
+category: Kickstarters
+comments: true
+---
+
+
+The Ordina Kickstarter trajectory is a collection of courses tailored and designed by the senior consultants of Ordina.
+These courses are created to give the beginning software developer a broad knowledge base while also providing an in-depth view on several technologies and best practices.
+
+This year the Kickstarter trajectory spanned 15 days, with topics ranging far and wide: backend to frontend, Spring Data JPA to TypeScript and everything in between.
+All of these courses will make sure the candidates will be able to hit the ground running on their first project as Ordina consultants.
+
+This post will summarize the training and experiences we've had while following the Kickstarter trajectory. 
+
+# Backlog
+
+<img src="/img/2018-03-02-Kickstarter-Trajectory-2018-light/kanban.jpg" style="max-width:100%"/>
+
+# What we have done
+
+### Git
+During the first days of our kickstart trajectory we've got a brief explanation about git by [Yannick De Turck](/author/yannick-de-turck/).
+He explained us that git is an open-source, distributed version control system that keeps track of your files and history.
+Basically this means git offers us tooling to collaborate on code bases without overwriting each others changes so easily.
+
+We saw which workflow JWorks uses in git and which commands we can use to do so.
+This way we learned how to create git repo's and create separate branches for features or different environments.
+And we even saw the different ways to merge these branches.
+One thing we'll definitely won't forget that fast is to rebase when pulling to keep a clean non-spaghetti like history,
+something that is preferred by many co-workers at JWorks! 
+At least that is what Yannick has told us ;)
+
+### Spring and Spring boot
+
+Those lectures were given by [Ken Coenen](/author/ken-coenen) and were spread over two days. 
+
+During the first lecture, we got a recap of the concepts of JPA, beans, application contexts and other things that Spring uses in its core fundamentals. Next, we dug deeper into the framework and introduced ourselves with Spring Web Services and Spring Security and created a small backend application during the second lecture. 
+
+Both lectures were rather theoretical, but very informative and elaborate with a lot of examples. So you have got everything you need to get familiar with Spring. You can find the course material on Ken's [Github](https://github.com/kencoenen/spring-course) page.
```

Comment:
Handy tip for links: append `{:target="_blank"}` to the end of it to open links in a new tab.
That way we make sure that the reader remains on our tech blog.

Example: `[Some linked resource](https://foo.com){:target="_blank"}`

Reply:
WIP